util.go

```diff
@@ -12,6 +14,18 @@ import (
 	"github.com/pion/transport/vnet"
 )
 
+func init() { // nolint:gochecknoinits
+	// Seeding random generator each time limits number of generated sequence to 31-bits,
+	// and causes collision on low time accuracy environments.
+	// Use global random generator seeded by crypto grade random.
+	var seed int64
+	if err := binary.Read(cryptorand.Reader, binary.LittleEndian, &seed); err != nil {
+		// crypto/rand is unavailable. Fallback to seed by time.
+		seed = time.Now().UnixNano()
+	}
+	rand.Seed(seed)
```

Comment:
To avoid touching `math/rand` global generator, we need to declare package global random generator in this package. But in this case, we need to be careful that all code using random must use this generator. If `math/rand` global generator is used, it will be seeded by default value all time. (might be potentially dangerous)

Reply:
Can we just not use `math/rand` instead? Not a big fan of `init`